assembly/pom.xml

```diff
@@ -1,4 +1,21 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Licensed to Cloudera, Inc. under one
```

Comment:
I'm just standardizing the style. Some existing headers have the leading `~`, some don't, I chose to use it.